src/implementation/Branch.ts

```diff
@@ -406,7 +414,26 @@ export class Branch {
 		return this.id;
 	}
 
-	getNode(...keys: string[]): RepoNodeWithData | RepoNodeWithData[] {
+	getNode<NodeData = Record<string, unknown>>({
```

Comment:
This is the breaking change in a public method, that might trigger a 2.0.0 release.
It's not in any of the public Enonic API functions though...